### PR TITLE
SRIS-1

### PR DESCRIPTION
**Pull request by dhlevi**
Correction for floating point conversion resulting in string values containing commas to return invalid results
See https://apps.nrs.gov.bc.ca/int/jira/browse/SRIS-1

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -249,6 +249,18 @@ def extract_floating_from_string(value):
...
+    """Convert string to Float, if possible"""
+    _result = 0
+    try:
+        if (value is not None and value != ''):
+            _exp_result = re.sub('[^\d\.]', '', value)
+            _result = (float(_exp_result))
+
+    except ValueError as _ve:
+        logging.exception(constant.VALUE_ERROR_EXCEPTION_RAISED, _ve)
+
```
Comment (mishraomp) [inline helper.py:263]: would it be better if `None` is returned than `0`

**Review by mishraomp — APPROVED** [review]
one minor comment, otherwise LGTM

File: helper.py (hunk 1 above)
Comment (dhlevi) [reply helper.py:263]: Yeah, fair point. I've update that function to default to `None`. Additionally, the `create_soil_volumes` section that's calling this has been updated to include a check for None and a `continue` so we don't get any unwanted math errors

**Comment by mishraomp** [comment]
@dhlevi Please merge when you can and deploy to PROD

**Pull request merged**